lib/const.rb

```diff
@@ -46,19 +46,21 @@
 # downloaded, e.g. in a docker container.
 ARCH_LIB = ARCH.eql?('x86_64') && Dir.exist?('/lib64') ? 'lib64' : 'lib'
 
+libc_path = `whereis libc.so.6`[11, 1000]
+
 # Glibc version can be found from the output of libc.so.6
-LIBC_VERSION = `/#{ARCH_LIB}/libc.so.6`[/Gentoo ([^-]+)/, 1]
+LIBC_VERSION = `#{libc_path}`[/Gentoo ([^-]+)/, 1]
```

Comment:
It's fine to leave this the way it is currently.

Reply:
This change is the point of the PR-- it lets crew be run on non ChromeOS distros-- very helpful for CI and testing.

Reply:
Is the location different on non ChromeOS distros?  I'm thinking probably not?  Do we have an example where it wouldn't be in the `ARCH_LIB` location?

Reply:
Debian

Reply:
Why would anyone in their right mind want to run crew on a Debian system?  You already have a superior package manager available.

Reply:
It's helpful for creating scripts and tests that use lib/const.rb, without needing to spin up a Docker instance.